tox.ini

```diff
@@ -60,46 +60,24 @@ deps =
 commands =
   charm: pyright
 
-[testenv:unit]
-description = Run unit tests
+[testenv:{unit, scenario}]
+description = Unit and scenario tests
 deps =
     pytest
-    coverage[toml]
-    -r{toxinidir}/requirements.txt
-commands =
-    coverage run --source={[vars]src_path} \
-                 -m pytest \
-                 --tb native \
-                 -v \
-                 -s \
-                 {posargs} \
-                 {[vars]tst_path}/unit
-    coverage report
-
-[testenv:scenario]
-description = Scenario tests
-deps =
-    coverage[toml]
-    pytest
+    pytest-cov
     ops-scenario>=7.0.1
     -r{toxinidir}/requirements.txt
 commands =
-    coverage run \
-      --source={[vars]src_path} \
-      -m pytest -v --tb native --log-cli-level=INFO -s {posargs} {[vars]tst_path}/scenario
-    coverage report
+    pytest {tty:--color=yes} --cov={[vars]src_path} --cov-config={tox_root}/pyproject.toml \
+        ;    for us
+        --cov-report=html:{tox_root}/results/html-cov/ \
+        ;    for tiobe
+        --cov-report=xml:{tox_root}/results/coverage-{env_name}.xml \
+        ;    for sparta
```

Comment:
Haha!
But seriously though, what is this for?

Reply:
not sure. copied it from charmcraft's coverage suite. they said tiobe needs it

Reply:
I thought the results had to go into a `cover` folder, do I remember wrong?

Reply:
apparently [this is how they're doing it in charmcraft](https://github.com/canonical/charmcraft/blob/main/tox.ini#L56) for the integration with tiobe, @simskij confirmed, although I can find absolutely zero documentation to confirm

Reply:
we'll see. it's little effort to change it back either way